### PR TITLE
refactor: remove workdir workaround

### PR DESCRIPTION
**Pull request by thesayyn**

### CODE REVIEW

File: MODULE.bazel [hunk 1]
```diff
@@ -11,7 +11,7 @@ bazel_dep(name = "bazel_skylib", version = "1.4.1")
 bazel_dep(name = "platforms", version = "0.0.5")
 
 oci = use_extension("//oci:extensions.bzl", "oci")
-oci.toolchains(crane_version = "v0.14.0")
+oci.toolchains(crane_version = "v0.15.1")
```
Comment (alexeagle) [inline MODULE.bazel:14]: are users forced to upgrade as well? (maybe we're forcing them here, if our version takes precedence over one that they register?

File: oci/private/image.sh.tpl [hunk 2]
```diff
@@ -79,13 +79,16 @@ EOF
     cat "${refs}"
 }
 
+function history() {
```
Comment (alexeagle) [inline oci/private/image.sh.tpl:82]: this seems to be unused

File: MODULE.bazel (hunk 1 above)
Comment (thesayyn) [reply MODULE.bazel:14]: yes, this is a breaking change if they register one of their own toolchain.

Comment (thesayyn) [reply MODULE.bazel]: We have already updated crane in another PR. While technically this is a breaking change, i don't expect any side effects from upgrading crane.

**Review by alexeagle — APPROVED** [review]

**Pull request merged**